src/portalbackend/PortalBackend.PortalEntities/PortalDbContext.cs

```diff
@@ -1301,5 +1303,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             entity.HasIndex(e => new { e.VerifiedCredentialExternalTypeId, e.Version })
                 .IsUnique();
         });
+        modelBuilder.Entity<CompaniesLinkedServiceAccount>()
+             .ToView(nameof(CompanyLinkedServiceAccounts))
```

Comment:
the .ToView method has an overload to specify the schema.

```suggestion
             .ToView(nameof(CompanyLinkedServiceAccounts), "portal")
```

Reply:
fixed